types.go

```diff
@@ -1247,6 +1278,185 @@ func (i *Float128) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// added 2022-11-02
+
+// Int
+// Experimental implementation
+
+// used big.Int
+// experimental type, and will replace or be replaced with previous Int implementation
+type Int struct {
```

Comment:
Quite unsure about this new type. I see it's used for `Decimal` only right now, and I'm against the change from `Float64` to `Decimal` higher in the PR. So for now, I would leave this aside until we see good reason to have them.

At which point, using an existing 256 bits integer library seems more appropriate, like https://github.com/holiman/uint256/blob/master/uint256.go which despite it's name supported Int and Uint. This is what `go-ethereum` use for their 256 bits math.

Reply:
Don't want to change or add duplicated types, but want to hear your thought - you prefer big.Int or not.

If yes, try to modify the current Int(might be?)
If not, roll back to current Int and Float

Reply:
My reasoning here is to align with whatever the canonical source code EOSIO/Leap platform uses. If it uses a `long`, then we should use a `int64`, if it uses `unsigned long`, we use `uint64`, etc. I don't see any reason to use `big.Int` when the EOSIO uses a fixed size type.

In the case of `Decimal`, I did not see the need for it yet. I've checked the system contract related to `VoterInfo` that you modified above to use `Decimal`, in the system contract, the value used there is a `double` (https://github.com/AntelopeIO/reference-contracts/blob/main/contracts/eosio.system/include/eosio.system/eosio.system.hpp#L302) so I don't see why using `Float64` would be wrong and what would be the gain of using `Decimal` here.

If there is something wrong with `Float64` in respect to for example how it serialize/deserialize according to EOSIO/Leap rules, I'm open to modify it.

Reply:
By https://github.com/eoscanada/eos-go/pull/191#discussion_r1016782712 , the the type will be removed

Reply:
@maoueh Any comment of them?